### PR TITLE
Add Humble Data as an event

### PR DESCRIPTION
**Pull request by cybit**
Text for the Humble Data workshop at Europython 2024.

Original author was Jodie Burchell.
Authored-by: Jodie Burchell

### CODE REVIEW

File: src/content/pages/humble-data.mdx [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+title: Humble Data Workshop
+subtitle: A workshop to teach beginners how to start with Python and Data Science.
+---
+
+# Humble Data Workshop
+Following a successful workshop at EuroPython last year, as well as events at PyCon US, Ghana, Namibia, Africa, Germany, Italy and PyData Global, we are delighted to be returning with the Humble Data workshop this year!
+
+👉🏽 [APPLY HERE TO ATTEND THIS WORKSHOP](https://forms.gle/jFP1CCNSp91ekyi16) (the application deadline is July 1st, 2024) 👈
```
Comment (hypha) [inline src/content/pages/humble-data.mdx:9]: This is 2023's form here.

**Review by hypha — CHANGES_REQUESTED** [review]

**Review by hypha — APPROVED** [review]

File: src/content/pages/humble-data.mdx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+We invite those from under-represented groups to apply to join us on **Tuesday 9th July 2024** for the Humble Data Workshop at EuroPython. There, you will learn the basics of programming in Python, useful libraries and tools, such as Jupyter Notebook, that help with data analysis.
+
+You will be divided into small groups, each with an assigned mentor to guide you through the workshop materials and answer your questions. You can expect plenty of exercises, as well as inspiring talks from those who were once in your shoes.
+## WHEN?
+Tuesday, 9th July 2024, 09:30 to 12:45.
+## WHERE?
+EuroPython (see [here](https://ep2024.europython.eu/where)).
+## WHO CAN JOIN?
+Anyone with a EuroPython [Conference Ticket or Combined Ticket](https://ep2023.europython.eu/tickets#ticket-types). Please note that you will not be able to participate without a ticket - thank you for your understanding!
```
Comment (egeakman) [inline src/content/pages/humble-data.mdx:23]:
```suggestion
Anyone with a EuroPython [Conference Ticket or Combined Ticket](https://ep2024.europython.eu/tickets#ticket-types). Please note that you will not be able to participate without a ticket - thank you for your understanding!
```

Bump year

Comment (egeakman) [reply src/content/pages/humble-data.mdx:23]: Also shouldn't we include Tutorial Tickets too?

File: src/content/pages/humble-data.mdx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+## Learn Python for Data Science
+Would you like to learn to code but don’t know where to start? Would you like to take a peek at Data Science but are not sure how? Taking the first steps is very hard. That is why we have decided to host a workshop to teach beginners how to start with Python and Data Science. Join and let's share our passion for the world of data science!
+
+“We want our community to be more diverse: whoever you are, and whatever your background, we welcome you.” – Python Software Foundation’s Diversity Statement
+
+We invite those from under-represented groups to apply to join us on **Tuesday 9th July 2024** for the Humble Data Workshop at EuroPython. There, you will learn the basics of programming in Python, useful libraries and tools, such as Jupyter Notebook, that help with data analysis.
+
+You will be divided into small groups, each with an assigned mentor to guide you through the workshop materials and answer your questions. You can expect plenty of exercises, as well as inspiring talks from those who were once in your shoes.
+## WHEN?
+Tuesday, 9th July 2024, 09:30 to 12:45.
```
Comment (cmaureir) [inline src/content/pages/humble-data.mdx:19]:
```suggestion
Tuesday, 9th July 2024, 09:30 to 17:00.
```
It's a full day now.

File: src/content/pages/humble-data.mdx [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+## WHERE?
+EuroPython (see [here](https://ep2024.europython.eu/where)).
```
Comment (cmaureir) [inline src/content/pages/humble-data.mdx:21]:
```suggestion
Prague Congress Centre (PCC), Room Club C

More information about the venue [here](https://ep2024.europython.eu/where).
```

File: src/content/pages/humble-data.mdx [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+Would you like to learn to code but don’t know where to start? Would you like to take a peek at Data Science but are not sure how? Taking the first steps is very hard. That is why we have decided to host a workshop to teach beginners how to start with Python and Data Science. Join and let's share our passion for the world of data science!
+
+“We want our community to be more diverse: whoever you are, and whatever your background, we welcome you.” – Python Software Foundation’s Diversity Statement
+
+We invite those from under-represented groups to apply to join us on **Tuesday 9th July 2024** for the Humble Data Workshop at EuroPython. There, you will learn the basics of programming in Python, useful libraries and tools, such as Jupyter Notebook, that help with data analysis.
+
+You will be divided into small groups, each with an assigned mentor to guide you through the workshop materials and answer your questions. You can expect plenty of exercises, as well as inspiring talks from those who were once in your shoes.
+## WHEN?
+Tuesday, 9th July 2024, 09:30 to 12:45.
+## WHERE?
```
[the remark this answers is not in the document]
Comment (cybit) [reply src/content/pages/humble-data.mdx:20]: See commit by Cristian.

File: src/content/pages/humble-data.mdx [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+👉🏽 Curiosity.
+
+👉🏽 Respect.
+
+👉🏽 Will to learn and to teach the mentees.
+
+In return, you will meet new interesting people outside your usual bubble, gain eternal gratitude from a group of attendees, and help to break down stereotypes around what a developer “looks like”.
+BONUS POINT: All this should be added on to your CV!
+
+Apply to be a mentor by [filling in this form](https://forms.gle/Xz9MTrFtRTwKDpSS8) by July 1st, 2024.
```
[the remark this answers is not in the document]
Comment (cybit) [reply src/content/pages/humble-data.mdx]: I made the mentor bold but left the rest. Does it look better in the newest version? @hypha

**Review by hypha — APPROVED** [review]

File: src/data/links.json [hunk 7]
```diff
@@ -55,7 +55,8 @@
         {
           "name": "C API Summit",
           "path": "/capi"
-        }
+        },
+        { "name": "Humble Data", "path": "/humble-data" }
```
Comment (egeakman) [inline src/data/links.json:59]:
```suggestion
        {
          "name": "Humble Data",
          "path": "/humble-data"
        }
```

**Review by egeakman — APPROVED** [review]
Thanks @cybit :heart:

No blockers, just small suggestions

File: src/data/links.json (hunk 7 above)
Comment (cybit) [reply src/data/links.json:59]: hmm, this was done by precommit on my system?

Comment (egeakman) [reply src/data/links.json:59]: Oh really? Well, we better keep it that way then, or the bot will bug us until eternity :D

File: src/content/pages/humble-data.mdx [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+## WHAT DO I NEED TO KNOW?
+If you have zero programming experience, this workshop is made for you! If you already have some experience but are new to Python, we’ve got you covered too – the materials are designed for attendees to progress through at their own pace.
+## HOW IS THIS POSSIBLE?
+This workshop is being run by data scientists and software engineers dedicated to the spirit of open source. Every day we benefit from the hard work of others when we use free open tools. This is our way of giving back. We thank EuroPython for their support in facilitating this workshop. ❤️
+
+If you would like to attend this workshop as a mentee
+
+<div class="text-center mb-8">
+<ButtonLink href="https://forms.gle/vJTYicVZVSeiRhgu9">Apply as a mentee now!</ButtonLink>
+
```
[the remark this answers is not in the document]
Comment (cybit) [reply src/content/pages/humble-data.mdx:44]: The original text had two areas to apply as mentee and one ad the end to apply as mentor.
No you want to remove the second - very visible - button for the mentee?

[the remark this answers is not in the document]
Comment (egeakman) [reply src/content/pages/humble-data.mdx:44]: No, just the extra space between the elements:

```diff
+ <div blabla>
+ <Component>
-
+ </div>

[the remark this answers is not in the document]
Comment (cybit) [reply src/content/pages/humble-data.mdx]: Sorry, that's not how I interpreted the suggestion. My bad. Did it now on my machine.

**Pull request merged**